spec/support/shared_examples/behaves_like_invitable.rb

```diff
@@ -0,0 +1,95 @@
+# shared examples for Invitable
+# Example usage:
+#  include_examples "Invitable", :workship_invitation, :workshop
+#
+RSpec.shared_examples 'Invitable' do |invitation_type, invitable_type|
+  let(:invitable) { Fabricate(invitable_type) }
+
+  context '#attendances' do
+    it 'permits accepted' do
+      invitation = Fabricate(invitation_type,
+                             invitable_type => invitable,
```

Comment:
can this be updated to `invitable_type: invitable` please

Reply:
I did initially try JSON style/Ruby 1.9 hash style and I got the following error.

```ruby
  Failure/Error:
       invitation = Fabricate(invitation_type,
                              member: Fabricate(:banned_member),
                              invitable_type: invitable,
                              attending: true)
     
     ActiveRecord::UnknownAttributeError:
       unknown attribute 'invitable_type' for WorkshopInvitation.
     Shared Example Group: "Invitable" called from ./spec/models/workshop_spec.rb:5
     # ./spec/support/shared_examples/behaves_like_invitable.rb:22:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:80:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # NoMethodError:
     #   undefined method `invitable_type=' for #<WorkshopInvitation:0x00007fa175f84eb8>
     #   ./spec/support/shared_examples/behaves_like_invitable.rb:22:in `block (3 levels) in <top (required)>'
```

I've done some reading around to try and understand this as best as I can (see references below). I haven't found anything authoritative [the following quote was the most interesting](https://stackoverflow.com/questions/8675206/is-there-any-difference-between-the-key-value-and-key-value-hash-no).

> The JavaScript style (key: value) is only useful if all of your Hash keys are "simple" symbols (more or less something that matches /\A[a-z_]\w*\z/i, AFAIK the parser uses its label pattern for these keys).

So, with the JSON style/Ruby 1.9 hash style is good for literal symbols - the colon at the end means it can convert it into symbol and use it as a hash key. In this case invitable_type is **not** a literal symbol it's a variable that holds a symbol, so I believe, it can't be converted into a symbol by the JSON style hash key implicitly - it thinks it a method and gives the above error.

There _may_ be ways around this and I've tried weird conversions that would look way worse the than hash rocket without success. 

Anyway, this is what I've tried. Do you know of a solution or would it be acceptable to leave it as a hash rocket?

### References
[Ruby docs Hash](https://ruby-doc.org/core-2.7.0/Hash.html)
[How to access a symbol hash key using a variable in Ruby](https://stackoverflow.com/questions/25537934/how-to-access-a-symbol-hash-key-using-a-variable-in-ruby)
[Is there any difference between the `:key => “value”` and `key: “value”` hash notations?](https://stackoverflow.com/questions/8675206/is-there-any-difference-between-the-key-value-and-key-value-hash-no)

Reply:
@despo I pressed the wrong button. I pressed resolved when I meant to press comment. (Github messaging isn't my strong point as I'm sure matyikriszta would attest to).

Reply:
I don't see it as resolved :)

However I'm happy with this explanation and will set all `key: value` comments I made to resolved.